app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  validates :first_name, presence: true, length: { maximum: 255 }
+  validates :last_name, presence: true, length: { maximum: 255 }
+  validates :email, presence: true, length: { maximum: 255 }
+  validates :password, presence: true, length: { minimum: 8, maximum: 255 }
```

Comment:
I assume in final version we'll change this out for a password hash so this is fine for now